front-end/src/screens/MenuBar/MobileMenu.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { ReactNode, useContext, useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Container, Menu, Icon, Sidebar, Sticky } from 'semantic-ui-react';
+import styled from '@xstyled/styled-components';
+
+import { UserDetailsContext } from '../../context/UserDetailsContext';
+import { useLogoutMutation } from '../../generated/auth-graphql';
+import { useRouter } from '../../hooks';
+import { logout } from '../../services/auth.service';
+
+interface Props {
+	children?: ReactNode,
+	className?: 'mobilemenu'
+}
+
+export const NavBarChildren = ({ children }:Props) => (
+	<Container>{children}</Container>
+);
+
+const MobileMenu = ({ children, className }:Props) => {
+	const [leftVisible, setLeftVisible] = useState(false);
+	const [rightVisible, setRightVisible] = useState(false);
+	const [pusherheight, setPusherheight] = useState('auto');
+
+	const handleLeftToggle = () => {
+		leftVisible ? setLeftVisible(false) : setLeftVisible(true);
+		rightVisible ? setRightVisible(false) : setRightVisible(false);
+		pusherheight === '100vH' && !rightVisible ? setPusherheight('auto') : setPusherheight('100vH');
+	};
+
+	const handleRightToggle = () => {
+		rightVisible ? setRightVisible(false) : setRightVisible(true);
+		leftVisible ? setLeftVisible(false) : setLeftVisible(false);
+		pusherheight === '100vH' && !leftVisible ? setPusherheight('auto') : setPusherheight('100vH');
+	};
+
+	const handleClose = () => {
+		setLeftVisible(false);
+		setRightVisible(false);
+		setPusherheight('auto');
```

Comment:
you are setting `auto` or `100vh` to the pusher, whereas the https://codesandbox.io/s/325y47xk36 you've linked is setting its visibility. Is there a reason for not using the visibility?

Reply:
Fixed navbar could very well be the root of this problem. Will try without it